docs/rules/jsx-quotes.md

```diff
@@ -19,21 +19,26 @@ If you want to have e.g. a double quote within a JSX attribute value, you have t
 
 ## Rule Details
 
-This rule takes one argument.
-If it is `"prefer-double"` then the rule enforces the usage of double quotes for all JSX attribute values which doesn’t contain a double quote.
-If `"prefer-single"` is configured then the rule enforces the usage of single quotes for all JSX attribute values which doesn’t contain a single quote.
+This rule enforces the consistent use of either double or single quotes in JSX attributes.
 
-The default is `"prefer-double"`.
+## Options
 
-The following patterns are considered problems when set to `"prefer-double"`:
+This rule has a string option:
+
+* `"prefer-double"` (default) enforces the usage of double quotes for all JSX attribute values that don't contain a double quote.
+* `"declaration"` enforces the usage of single quotes for all JSX attribute values that don’t contain a single quote.
```

Comment:
s/declaration/prefer-single/
optional on this and the preceding line: s/usage/use/